### PR TITLE
chore(Icons): rename expand to maximize and collapse to minimize

### PR DESCRIPTION
**Pull request by shridhar52war**

### CODE REVIEW

File: src/icons/Maximize/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export { default } from './Maximize';
```
Comment (dhruvdutt) [inline src/icons/Maximize/index.js:1]: Rename components from Expand to Maximize inside Maximize.js

Comment (kamleshchandnani) [reply src/icons/Maximize/index.js:1]: @shridhar52war you did it on `minimize` but forgot in `maximize` 😆

Comment (shridhar52war) [reply src/icons/Maximize/index.js:1]: 🤦

File: src/icons/index.js [hunk 2]
```diff
@@ -10,11 +10,11 @@ const icons = {
   chevronLeft: require('./ChevronLeft').default,
   chevronRight: require('./ChevronRight').default,
   chevronUp: require('./ChevronUp').default,
-  collapse: require('./Collapse').default,
   close: require('./Close').default,
-  expand: require('./Expand').default,
   info: require('./Info').default,
   invoice: require('./Invoice').default,
+  maximize: require('./Maximize').default,
```
Comment (dhruvdutt) [inline src/icons/index.js:16]: Thanks for sorting. 💯

**Pull request merged**